modules/siddhi-extensions/time/src/test/java/org/wso2/siddhi/extension/time/ExtractAttributesFunctionExtensionTestCase.java

```diff
@@ -596,7 +615,6 @@ public void receive(long timeStamp, Event[] inEvents, Event[] removeEvents) {
                 for (Event inEvent : inEvents) {
                     count++;
                     log.info("Event : " + count + ",WEEK : " + inEvent.getData(1));
-                    Assert.assertEquals(41, inEvent.getData(1));
```

Comment:
Why Assertion is removed ?

Reply:
Thanks for letting me know about the issue, here we cannot guarantee the output results from siddhi when we just pass the time value to time extract extension.So, we just assert only whether we get the event for the given query.

But as you pointed asserting value is also important, I do the similar time formatting we do in the extension before executing the test case and use it as expected value in the assertion.

Please find the relevant commit ID for this fixes: e75fe206ff0dd0f40afef0674685c57ef5b15c6b